test/libsolidity/semanticTests/strings.sol

```diff
@@ -0,0 +1,8 @@
+contract C {
+    string public shortString = "any";
+    string public longString = "anyanyanyanyanyanyanyanyanyanyanyanyanyanyanyany";
+}
+// ====
+// EVMVersion: >=constantinople
```

Comment:
Why constantinople?

Reply:
This will be removed. The test will be part of the smoke tests.